File: src/main/java/com/grepp/curdsample/dto/TaskPageDto.java
package com.grepp.curdsample.dto;

import lombok.Builder;
import lombok.Data;

import java.util.List;

@Data
@Builder
//public record TaskPageDto(boolean hasNext, List<TaskDto> tasks) {
public class TaskPageDto {

    private boolean hasNext;
    private List<TaskDto> data;

}

File: src/main/java/com/est/crudsample/dto/TaskPageDto.kt
package com.est.crudsample.dto

data class TaskPageDto(
    val hasNext: Boolean,
    val data: List<TaskDto>,
)


//@Data
//@Builder
//public class TaskPageDto {
//
//    private boolean hasNext;
//    private List<TaskDto> data;
//
//}

